src/System.Data.SqlClient/src/System/Data/SqlClient/SqlClientDiagnosticListenerExtensions.cs

```diff
@@ -0,0 +1,73 @@
+﻿using System.Collections;
+using System.Diagnostics;
+
+namespace System.Data.SqlClient
+{
+    /// <summary>
+    /// Extension methods on the DiagnosticListener class to log SqlCommand data
+    /// </summary>
+    internal static class SqlClientDiagnosticListenerExtensions
+    {
+        public static readonly string DiagnosticListenerName = "SqlClientDiagnosticListener";
+
+        private static readonly string SqlClientPrefix = "System.Data.SqlClient.";
+        public static readonly string SqlBeforeExecuteCommand = SqlClientPrefix + nameof(WriteCommandBefore);
+        public static readonly string SqlAfterExecuteCommand = SqlClientPrefix + nameof(WriteCommandAfter);
+        public static readonly string SqlErrorExecuteCommand = SqlClientPrefix + nameof(WriteCommandError);
```

Comment:
@justinvp I have made these changes and they are reflected in commit https://github.com/dotnet/corefx/pull/8420/commits/319736ba1954415602e7aecf25ba52daee275f36 Thanks! :smile: